src/modules/IPSecConf.rb

```diff
@@ -129,19 +129,19 @@ def GenFirewallScript
             script << func_template % {func_name: "fw_custom_before_port_handling", content: ""}
             # Reduce TCP MSS - if this has to be done, it must come before FORWARD and MASQUERADE
             inet_access = ""
-            if @tcp_mss_1024
-                inet_access += "iptables -A FORWARD -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --set-mss 1024\n" +
-                               "ip6tables -A FORWARD -p tcp --tcp-flags SYN,RST SYN -j TCPMSS --set-mss 1024\n"
+            if @tcp_reduce_mss
+                inet_access += "iptables -A FORWARD -p tcp -m tcp --tcp-flags SYN,RST SYN -m tcpmss --mss 1221:65535 -j TCPMSS --set-mss 1220\n" +
+                               "ip6tables -A FORWARD -p tcp -m tcp --tcp-flags SYN,RST SYN -m tcpmss --mss 1221:65535 -j TCPMSS --set-mss 1220\n"
             end
             # Forwarding for Internet access
-            forward_template = "%s -A FORWARD -s %s -j ACCEPT\n"
+            forward_template = "%s -A FORWARD -s %s -j ACCEPT\n%s -A FORWARD -d %s -j ACCEPT\n"
```

Comment:
`"%{iptables} -A FORWARD -s %{cidr} -j ACCEPT\n%{iptables} -A FORWARD -d %{cidr} -j ACCEPT\n"` ?
But the bigger problem is separating the format string from its application, that is hard to read.